runtime/queries/awk/highlights.scm

```diff
@@ -47,21 +47,21 @@
 (number) @number
 
 (string) @string
-(regex) @string.regex
+(regex) @string.regexp
 (escape_sequence) @string.escape
 
 (comment) @comment @spell
 
-((program . (comment) @preproc)
-  (#lua-match? @preproc "^#!/"))
+((program . (comment) @keyword.directive)
+  (#lua-match? @keyword.directive "^#!/"))
```

Comment:
_That_ is the only kind of change (direct consequence of capture renaming) that's in scope here. (And the reason why it's better to review by comment rather than by file.)

I'd appreciate a list of patterns that should be changed; it's not always obvious what's a shebang and what not.

Personally, I'm not sure I agree, though; it feels more like a directive to me (and Helix uses `@keyword`, too).

Reply:
> I'd appreciate a list of patterns that should be changed; it's not always obvious what's a shebang and what not.

Any `@preproc` comments.

Reply:
so any `@keyword.directive` that applies to a `(comment)` node?

Reply:
Yeah.

Reply:
Also any `shebang` / `hashbang` nodes that some languages have.